core/docs/spark-qualification-tool.md

```diff
@@ -628,6 +629,15 @@ The following table lists the exec's name and the status of parsing their expres
 | WindowExec                            |              -              |         x        |        -        |
 | WindowInPandasExec                    |              -              |         -        |        x        |
 
+### MLFunctions report
+The Qualification tool generates a report if there are SparkML or Spark XGBoost functions used in the eventlog.
+The functions in "*spark.ml.*" or "*spark.XGBoost.*" packages are displayed in the report.
+
+1. _App ID_
+2. _Stage ID_
+3. _Ml Functions_: List of ML functions used in the corresponding stage.
+4. _Stage Task Duration_: amount of time spent in tasks of SQL Dataframe operations for the given stage.
```

Comment:
I assume this is not  time spent in tasks of SQL dataframe operations, but rather in stages containing ML functions?

Reply:
My bad. Forgot to update it after copying it from other section. PTAL.